cmd/_integration-tests/transport/http_test.go

```diff
@@ -158,6 +158,26 @@ func TestCtxToCtxViaHTTPHeaderClient(t *testing.T) {
 	}
 }
 
+func TestEchoOddNamesClient(t *testing.T) {
```

Comment:
We should get in the habit of putting a small comment on why a test was added. Just so we can see in the future while scrolling through.

Please either make a github issue about this and reference it or add a small comment like:

```
// Test different kinds of message field names from protobuf definition.
// e.g. "CamelCase", "snake_case", "__why_so_many_underscores"
```